index.html

```diff
@@ -287,6 +315,13 @@
     adjustFontSize();
     window.addEventListener("orientationchange", () => { setTimeout(adjustFontSize, 100) });
 
+    // グローバルにコンテキストメニューとテキスト選択の開始を抑制（初期化時に一度だけ追加）
+    document.addEventListener('contextmenu', (e) => e.preventDefault());
+    // グローバルにコンテキストメニューとテキスト選択の開始を抑制
+    // （必要な操作がある場合は限定的にする）
+    document.addEventListener('contextmenu', (e) => e.preventDefault());
+    document.addEventListener('selectstart', (e) => e.preventDefault());
```

Comment:
ご対応ありがとうございます！

iOSでの動作確認をしていなかったため、ご指摘いただき助かりました。
差分を見る限り、当該の問題は修正されているものと拝見しました。

1点、お願いがあります。
318〜322行目で contextmenu イベントリスナーが重複して登録されています。
以下のように、1つにまとめていただけますと幸いです。

```suggestion
    // グローバルにコンテキストメニューとテキスト選択の開始を抑制
    // （必要な操作がある場合は限定的にする）
    document.addEventListener('contextmenu', (e) => e.preventDefault());
    document.addEventListener('selectstart', (e) => e.preventDefault());
```

動作には影響しませんが、コードの整理のためご対応いただけると助かります 🙇

Reply:
編集しておきます！ご指摘ありがとうございます！